### ISSUE TITLE
Snarl doesn't like being passed two consecutive colons from certain APIs

### ISSUE THREAD

**Issue by blackwind** [fullphat/snarl#13]
It seems Snarl chokes on two consecutive colons when passed via JSON or the Python API, while [Snarl_CMD.exe](http://tlhan-ghun.de/projects/snarl-command-line-tools/snarl_cmd-exe/) accepts them just fine.

Example:

    http://localhost:9889/?d={%22action%22:2,%22payload%22:%22notify?title=Test%3A%3ATest%22}

This produces a blank notification, when it should produce one with the title "Test::Test".

**Comment by fullphat** [comment]
Yes, this is an issue with the way snarl handles notification requests internally.  I need to do some serious re-plumbing of things.  Will try to find a workaround in the meantime.

Thanks 

Chris
full phat products
__________________________
http://getsnarl.info
Twitter: @snarlhappens

**Comment by fullphat** [comment]
This should be resolved in R5.0 but I'll check before closing the ticket.
